lib/cluster-providers/ec2-cluster-provider.ts

```diff
@@ -84,8 +84,15 @@ export class EC2ClusterProvider implements ClusterProvider {
     readonly options: EC2ProviderClusterProps;
 
 
-    constructor(options?: EC2ProviderClusterProps) {
-        this.options = options ?? { version: KubernetesVersion.V1_19 };
+    constructor(scope?: Construct, options?: EC2ProviderClusterProps) {
```

Comment:
`createCluster` already has scope passed in, it appears redundant to do it here as well. if you need to derive a reasonable default, let's do it in the `createCluster` method.